### ISSUE TITLE
About the training code for ip-adapter-plus-face_sdxl

### ISSUE THREAD

**Issue by fanghao123-qw** [tencent-ailab/IP-Adapter#202]
would like to inquire whether the training code for ip-adapter-plus-face_sdxl has been released. Is tutorial_train_sdxl.py the training code for ip-adapter-plus-face_sdxl? If not, what modifications should I make? How should I construct the dataset?"

**Comment by xiaohu2015** [comment]
ref to https://github.com/tencent-ailab/IP-Adapter/issues/198

I will update this week.

**Comment by fanghao123-qw** [comment]
Thank you for your response. I look forward to your work. 

> ref to #198
> 
> I will update this week.

Thank you for your response. I look forward to your work. I hope you can explain how to construct the training dataset. I can prepare it in advance, so when your code is released, I can give it a try.

**Comment by xiaohu2015** [comment]
> Thank you for your response. I look forward to your work.
> 
> > ref to #198
> > I will update this week.
> 
> Thank you for your response. I look forward to your work. I hope you can explain how to construct the training dataset. I can prepare it in advance, so when your code is released, I can give it a try.

dataset: https://github.com/tencent-ailab/IP-Adapter/issues/180

**Comment by vonchenplus** [comment]
Could you please provide the training parameters? What are the batch size and learning rate? Is the dimension of LoRA 128? I feel it's very easy to run out of VRAM, even with an A100.
